### ISSUE TITLE
No `master-latest-daily` release available

### ISSUE THREAD

**Issue by c0d1ngm0nk3y** [istio/istio#15150]
On `gcr.io/istio-release`, there is no daily release of the master available.

**Affected product area (please put an X in all that apply)**

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[X] Developer Infrastructure

**Expected behaviour**

There should be such a release every day.

Note: The images for all releases (`1.0`,`1.1`,`1.2`) are there. "Just" `master` is missing. So this might be related to https://github.com/istio/istio/pull/14737 which just got merged. Or is this just a coincidence.

How to get any information what went wrong in the release process?

**Comment by c0d1ngm0nk3y** [comment]
@utka I have tried (I am still trying) to get a local release to check if this is related to the whole `distroless` change. I have seen you have created a `local.sh` for purpose. Thanks. Until now, I am struggling with this script and maybe you are interested in issues:
* The directory `istio/output/local` did not exist the first time, so the `rm -rf` failed
* The script several times because I have not the necessary repos cloned (e.g. proxy or tools)
* The was a problem with some check, so I changed `CB_VERIFY_CONSISTENCY=false`
* `create_linux_archive` failed on my mac because `--owner` is not a valid option for the installed `tar`
